.github/workflows/release.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright © 2022 Kubernetes Authors
```

Comment:
```suggestion
# Copyright © 2023 Kubernetes Authors
```

Reply:
Hey @robscott,I think we can test it in the fork repo, after the previous PRs get merged refer to https://github.com/kubernetes-sigs/ingress2gateway/issues/21#issuecomment-1484915839.